open-hackathon-server/src/hackathon/hack/admin_manager.py

```diff
@@ -86,24 +89,30 @@ def get_entitled_hackathons_simple(self, user_id):
         :return list of hackathon simple
         """
 
-        admin_user_hackathon_simple = self.db.session().query(
-            Hackathon.id,
-            Hackathon.name,
-            Hackathon.display_name,
-            Hackathon.ribbon,
-            Hackathon.short_description,
-            Hackathon.banners,
-            Hackathon.status,
-            Hackathon.creator_id,
-            Hackathon.type,
-            (func.unix_timestamp(Hackathon.event_start_time)*1000).label('event_start_time'),
-            (func.unix_timestamp(Hackathon.event_end_time)*1000).label('event_end_time')
-        ).join(AdminHackathonRel, AdminHackathonRel.hackathon_id == Hackathon.id or AdminHackathonRel.hackathon_id == -1)\
-            .filter(AdminHackathonRel.user_id == user_id)\
-            .order_by(Hackathon.event_start_time.desc())\
-            .all()
-
-        return [h._asdict() for h in admin_user_hackathon_simple]
+        # admin_user_hackathon_simple = self.db.session().query(
+        #     Hackathon.id,
+        #     Hackathon.name,
+        #     Hackathon.display_name,
+        #     Hackathon.ribbon,
+        #     Hackathon.short_description,
+        #     Hackathon.banners,
+        #     Hackathon.status,
+        #     Hackathon.creator_id,
+        #     Hackathon.type,
+        #     (func.unix_timestamp(Hackathon.event_start_time)*1000).label('event_start_time'),
+        #     (func.unix_timestamp(Hackathon.event_end_time)*1000).label('event_end_time')
+        # ).join(AdminHackathonRel, AdminHackathonRel.hackathon_id == Hackathon.id or AdminHackathonRel.hackathon_id == -1)\
+        #     .filter(AdminHackathonRel.user_id == user_id)\
+        #     .order_by(Hackathon.event_start_time.desc())\
+        #     .all()
+
+        # return [h._asdict() for h in admin_user_hackathon_simple]
+
+        admin_user_hackathon_simple = Hackathon.objects(creator_id=user)\
```

Comment:
Please check `user.is_super()` first. Return all hackathons if `True`. Otherwise return the results as your code